### DISCUSSION TITLE
[Bug report]  forum software hacked ?

### DISCUSSION THREAD

**Discussion by jasonbrown1965** [GetPublii/Publii#1755] [Bug Report]
### Operating system

n/a

### Publii version

n/a

### Issue type

Something else

### Bug description

Was looking up error posts for SFTP on Publii, and when I finished reading one, clicked on the main forum button and it brought up an ad for a polish hosting company. Not what I was expecting on an English language site!

Above my paygrade, but community member on [VirusTotal](https://www.virustotal.com/graph/g81809a04aa944d1c9544cc1c5caf61643fa1cc014ae949718cae6379c1768555) thinks the .pl site hosts malware?

If it's a hack, I'm guessing that's because of the forum software, not Publii. 

Suggest:

- getting rid of our polish friends :)
- replacing poles with forum retired notice / re?direct link to discussions here
- notice on all old forum pages that discussions moved 

![image](https://github.com/user-attachments/assets/e929ce67-e265-4a20-8f8d-7991a9819cf0)

### Steps to reproduce

Visit :
https://forum.getpublii.com/

### Relevant log output

```shell
n/a
```

**Comment by bjazmoore** [comment]
Clear your cache and try again..  The forum is here: https://github.com/GetPublii/Publii/discussions

**Comment by bobmitro** [comment]
Ah, I see the issue. It's a temporary placeholder from our new hosting provider. I made a mistake during the file transfer; not everything was uploaded correctly. Thanks for bringing this to my attention.
